includes/class-newspack-newsletters-logger.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Newspack Newsletters Logger
+ *
+ * @package Newspack
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Logger.
+ */
+class Newspack_Newsletters_Logger {
+	/**
+	 * A logger.
+	 *
+	 * @param any $payload The payload to log.
+	 */
+	public static function log( $payload ) {
+		if ( ! defined( 'NEWSPACK_LOG_LEVEL' ) || 0 > (int) NEWSPACK_LOG_LEVEL ) {
+			return;
+		}
+
+		$header = 'NEWSPACK-NEWSLETTERS';
+		if ( class_exists( '\Newspack\Logger' ) ) {
+			\Newspack\Logger::log( $payload, $header );
+		} else {
+			error_log( '[' . $header . ']: ' . $payload ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
```

Comment:
If `$payload` is of `any` type, we should cast it to string before `error_log()`.

Reply:
Added type checking in 9b07f9c3ddcbbdbc76c48d0ebf559a9f21a8076f. To be extra safe, it'll exit if something else than a string is to be logged.